### DISCUSSION TITLE
Postal

### DISCUSSION THREAD

**Discussion by youngmike0** [postalserver/postal#2025] [Help with using Postal]
Usage: postal [-m maximum-message-size] [-M minimum-message-size] [-t threads]
              [-c messages-per-connection] [-r messages-per-minute] [-a]
              [-b [no]netscape] [-p port] [-[z|Z] debug-file]
              [-s ssl-percentage]
              [-L] [-l local-address] [-f sender-file]
              smtp-server user-list-filename

can someone help me get past this issue please

**Comment by willpower232** [comment]
I'm not familiar with that help text, are you definitely using this postal?

**Comment by youngmike0** [comment]
Yes this is where I am having issues..it's not allow me to put in admin -
user to finally get the app.. Is there another way to do this please ?
Thank You for your help ...
Michael

**Reply by willpower232** [reply]
Normally the command to make the initial user is `postal make-user` and then you can log into the web UI but as I said, I'm not sure which Postal command you have.
